[BUG]PPL queries fail on spark

**What is the bug?**
Running PPL query fails on spark (EMR) 

**How can one reproduce the bug?**
Run the next query in the workbench 
```
source = mys3.default.http_logs head 5
```
result with the next error respond:
```
{
    "data": {
        "ok": true,
        "resp": {
            "status": "FAILED",
            "error": "Syntax error: \nSyntax error at or near 'source'(line 1, pos 0)\n\n== SQL ==\nsource = mys3.default.http_logs head 5\n^^^\n"
        }
    }
}
```
EMR error-log
```
Syntax error at or near 'source'(line 1, pos 0)

== SQL ==
source = mys3.default.http_logs head 5
^^^

org.apache.spark.sql.catalyst.parser.ParseException: 
Syntax error at or near 'source'(line 1, pos 0)

== SQL ==
source = mys3.default.http_logs head 5
^^^

	at org.apache.spark.sql.catalyst.parser.ParseException.withCommand(ParseDriver.scala:306) ~[spark-catalyst_2.12-3.3.2-amzn-0.jar:3.3.2-amzn-0]
	at org.apache.spark.sql.catalyst.parser.AbstractSqlParser.parse(ParseDriver.scala:143) ~[spark-catalyst_2.12-3.3.2-amzn-0.jar:3.3.2-amzn-0]
	at org.apache.spark.sql.execution.SparkSqlParser.parse(SparkSqlParser.scala:52) ~[spark-sql_2.12-3.3.2-amzn-0.jar:3.3.2-amzn-0]
	at org.apache.spark.sql.catalyst.parser.AbstractSqlParser.parsePlan(ParseDriver.scala:89) ~[spark-catalyst_2.12-3.3.2-amzn-0.jar:3.3.2-amzn-0]
	at org.opensearch.flint.spark.sql.FlintSparkSqlParser.$anonfun$parsePlan$1(FlintSparkSqlParser.scala:64) ~[org.opensearch_opensearch-spark-standalone_2.12-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at org.opensearch.flint.spark.sql.FlintSparkSqlParser.parse(FlintSparkSqlParser.scala:104) ~[org.opensearch_opensearch-spark-standalone_2.12-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at org.opensearch.flint.spark.sql.FlintSparkSqlParser.parsePlan(FlintSparkSqlParser.scala:56) ~[org.opensearch_opensearch-spark-standalone_2.12-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at org.apache.spark.sql.SparkSession.$anonfun$sql$2(SparkSession.scala:620) ~[spark-sql_2.12-3.3.2-amzn-0.jar:3.3.2-amzn-0]
	at org.apache.spark.sql.catalyst.QueryPlanningTracker.measurePhase(QueryPlanningTracker.scala:192) ~[spark-catalyst_2.12-3.3.2-amzn-0.jar:3.3.2-amzn-0]
	at org.apache.spark.sql.SparkSession.$anonfun$sql$1(SparkSession.scala:620) ~[spark-sql_2.12-3.3.2-amzn-0.jar:3.3.2-amzn-0]
	at org.apache.spark.sql.SparkSession.withActive(SparkSession.scala:779) ~[spark-sql_2.12-3.3.2-amzn-0.jar:3.3.2-amzn-0]
	at org.apache.spark.sql.SparkSession.sql(SparkSession.scala:617) ~[spark-sql_2.12-3.3.2-amzn-0.jar:3.3.2-amzn-0]
	at org.apache.spark.sql.FlintJobExecutor.executeQuery(FlintJobExecutor.scala:317) ~[org.opensearch_opensearch-spark-sql-application_2.12-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at org.apache.spark.sql.FlintJobExecutor.executeQuery$(FlintJobExecutor.scala:309) ~[org.opensearch_opensearch-spark-sql-application_2.12-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at org.apache.spark.sql.FlintREPL$.executeQuery(FlintREPL.scala:42) ~[org.opensearch_opensearch-spark-sql-application_2.12-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at org.apache.spark.sql.FlintREPL$.$anonfun$executeQueryAsync$1(FlintREPL.scala:458) ~[org.opensearch_opensearch-spark-sql-application_2.12-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659) ~[scala-library-2.12.15.jar:?]
	at scala.util.Success.$anonfun$map$1(Try.scala:255) ~[scala-library-2.12.15.jar:?]
	at scala.util.Success.map(Try.scala:213) ~[scala-library-2.12.15.jar:?]
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292) ~[scala-library-2.12.15.jar:?]
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33) ~[scala-library-2.12.15.jar:?]
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33) ~[scala-library-2.12.15.jar:?]
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64) ~[scala-library-2.12.15.jar:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539) ~[?:?]
	at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
	at java.lang.Thread.run(Thread.java:833) ~[?:?]
```

**Sprak Submit Context**
```
 --class org.apache.spark.sql.FlintREPL  --conf spark.hadoop.fs.s3.customAWSCredentialsProvider=com.amazonaws.emr.AssumeRoleAWSCredentialsProvider  --conf spark.hadoop.aws.catalog.credentials.provider.factory.class=com.amazonaws.glue.catalog.metastore.STSAssumeRoleSessionCredentialsProviderFactory  --conf spark.jars.packages=org.opensearch:opensearch-spark-standalone_2.12:0.1.0-SNAPSHOT,org.opensearch:opensearch-spark-sql-application_2.12:0.1.0-SNAPSHOT,org.opensearch:opensearch-spark-ppl_2.12:0.1.0-SNAPSHOT  --conf spark.jars.repositories=https://aws.oss.sonatype.org/content/repositories/snapshots  --conf spark.emr-serverless.driverEnv.JAVA_HOME=/usr/lib/jvm/java-17-amazon-corretto.x86_64/  --conf spark.executorEnv.JAVA_HOME=/usr/lib/jvm/java-17-amazon-corretto.x86_64/  --conf spark.datasource.flint.host=ec2-18-237-133-156.us-west-2.compute.amazonaws.com  --conf spark.datasource.flint.port=9200  --conf spark.datasource.flint.scheme=http  --conf spark.datasource.flint.auth=noauth  --conf spark.datasource.flint.customAWSCredentialsProvider=com.amazonaws.emr.AssumeRoleAWSCredentialsProvider  --conf spark.sql.extensions=org.opensearch.flint.spark.FlintSparkExtensions,org.opensearch.flint.spark.FlintPPLSparkExtensions  --conf spark.hadoop.hive.metastore.client.factory.class=com.amazonaws.glue.catalog.metastore.AWSGlueDataCatalogHiveClientFactory  --conf spark.emr-serverless.driverEnv.ASSUME_ROLE_CREDENTIALS_ROLE_ARN=arn:aws:iam::924196221507:role/FlintOpensearchServiceRole  --conf spark.executorEnv.ASSUME_ROLE_CREDENTIALS_ROLE_ARN=arn:aws:iam::924196221507:role/FlintOpensearchServiceRole  --conf spark.hive.metastore.glue.role.arn=arn:aws:iam::924196221507:role/FlintOpensearchServiceRole  --conf spark.sql.catalog.mys3=org.opensearch.sql.FlintDelegatingSessionCatalog  --conf spark.flint.datasource.name=mys3  --conf spark.flint.job.requestIndex=.query_execution_request_mys3  --conf spark.flint.job.sessionId=NWpKUFBobmtiZ215czM= --conf spark.dynamicAllocation.enabled=false --conf spark.executor.instances=10
```
Replacing the extension list order for `--conf spark.sql.extensions` where setting ppl to be the first and resubmit the job:
`--conf spark.sql.extensions=org.opensearch.flint.spark.FlintPPLSparkExtensions,org.opensearch.flint.spark.FlintSparkExtensions`

produces the following:
```
23/10/27 06:06:12 INFO FlintREPL: read from .query_execution_request_mys3
23/10/27 06:06:13 ERROR FlintREPL: Unexpected error: java.net.ConnectException: Timeout connecting to [ec2-18-237-133-156.us-west-2.compute.amazonaws.com/18.237.133.156:9200]
java.lang.RuntimeException: java.net.ConnectException: Timeout connecting to [ec2-18-237-133-156.us-west-2.compute.amazonaws.com/18.237.133.156:9200]
	at org.opensearch.flint.core.storage.OpenSearchReader.hasNext(OpenSearchReader.java:48) ~[org.opensearch_opensearch-spark-standalone_2.12-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at org.apache.spark.sql.FlintREPL$.processCommands(FlintREPL.scala:280) ~[org.opensearch_opensearch-spark-sql-application_2.12-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at org.apache.spark.sql.FlintREPL$.queryLoop(FlintREPL.scala:156) ~[org.opensearch_opensearch-spark-sql-application_2.12-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at org.apache.spark.sql.FlintREPL$.main(FlintREPL.scala:99) ~[org.opensearch_opensearch-spark-sql-application_2.12-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at org.apache.spark.sql.FlintREPL.main(FlintREPL.scala) ~[org.opensearch_opensearch-spark-sql-application_2.12-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at org.apache.spark.deploy.JavaMainApplication.start(SparkApplication.scala:52) ~[spark-core_2.12-3.3.2-amzn-0.jar:3.3.2-amzn-0]
	at org.apache.spark.deploy.SparkSubmit.org$apache$spark$deploy$SparkSubmit$$runMain(SparkSubmit.scala:1006) ~[spark-core_2.12-3.3.2-amzn-0.jar:3.3.2-amzn-0]
	at org.apache.spark.deploy.SparkSubmit.doRunMain$1(SparkSubmit.scala:180) ~[spark-core_2.12-3.3.2-amzn-0.jar:3.3.2-amzn-0]
	at org.apache.spark.deploy.SparkSubmit.submit(SparkSubmit.scala:203) ~[spark-core_2.12-3.3.2-amzn-0.jar:3.3.2-amzn-0]
	at org.apache.spark.deploy.SparkSubmit.doSubmit(SparkSubmit.scala:90) ~[spark-core_2.12-3.3.2-amzn-0.jar:3.3.2-amzn-0]
	at org.apache.spark.deploy.SparkSubmit$$anon$2.doSubmit(SparkSubmit.scala:1095) ~[spark-core_2.12-3.3.2-amzn-0.jar:3.3.2-amzn-0]
	at org.apache.spark.deploy.SparkSubmit$.main(SparkSubmit.scala:1104) ~[spark-core_2.12-3.3.2-amzn-0.jar:3.3.2-amzn-0]
	at org.apache.spark.deploy.SparkSubmit.main(SparkSubmit.scala) ~[spark-core_2.12-3.3.2-amzn-0.jar:3.3.2-amzn-0]
Caused by: java.net.ConnectException: Timeout connecting to [ec2-18-237-133-156.us-west-2.compute.amazonaws.com/18.237.133.156:9200]
	at org.opensearch.client.RestClient.extractAndWrapCause(RestClient.java:953) ~[org.opensearch_opensearch-spark-standalone_2.12-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at org.opensearch.client.RestClient.performRequest(RestClient.java:332) ~[org.opensearch_opensearch-spark-standalone_2.12-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at org.opensearch.client.RestClient.performRequest(RestClient.java:320) ~[org.opensearch_opensearch-spark-standalone_2.12-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at org.opensearch.client.RestHighLevelClient.internalPerformRequest(RestHighLevelClient.java:1911) ~[org.opensearch_opensearch-spark-standalone_2.12-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at org.opensearch.client.RestHighLevelClient.performRequest(RestHighLevelClient.java:1877) ~[org.opensearch_opensearch-spark-standalone_2.12-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at org.opensearch.client.RestHighLevelClient.performRequestAndParseEntity(RestHighLevelClient.java:1845) ~[org.opensearch_opensearch-spark-standalone_2.12-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at org.opensearch.client.RestHighLevelClient.search(RestHighLevelClient.java:1088) ~[org.opensearch_opensearch-spark-standalone_2.12-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at org.opensearch.flint.core.storage.OpenSearchScrollReader.search(OpenSearchScrollReader.java:45) ~[org.opensearch_opensearch-spark-standalone_2.12-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at org.opensearch.flint.core.storage.OpenSearchReader.hasNext(OpenSearchReader.java:41) ~[org.opensearch_opensearch-spark-standalone_2.12-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	... 16 more
Caused by: java.net.ConnectException: Timeout connecting to [ec2-18-237-133-156.us-west-2.compute.amazonaws.com/18.237.133.156:9200]
	at org.apache.http.nio.pool.RouteSpecificPool.timeout(RouteSpecificPool.java:169) ~[org.opensearch_opensearch-spark-standalone_2.12-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at org.apache.http.nio.pool.AbstractNIOConnPool.requestTimeout(AbstractNIOConnPool.java:632) ~[org.opensearch_opensearch-spark-standalone_2.12-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at org.apache.http.nio.pool.AbstractNIOConnPool$InternalSessionRequestCallback.timeout(AbstractNIOConnPool.java:898) ~[org.opensearch_opensearch-spark-standalone_2.12-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at org.apache.http.impl.nio.reactor.SessionRequestImpl.timeout(SessionRequestImpl.java:198) ~[org.opensearch_opensearch-spark-standalone_2.12-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at org.apache.http.impl.nio.reactor.DefaultConnectingIOReactor.processTimeouts(DefaultConnectingIOReactor.java:213) ~[org.opensearch_opensearch-spark-standalone_2.12-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at org.apache.http.impl.nio.reactor.DefaultConnectingIOReactor.processEvents(DefaultConnectingIOReactor.java:158) ~[org.opensearch_opensearch-spark-standalone_2.12-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at org.apache.http.impl.nio.reactor.AbstractMultiworkerIOReactor.execute(AbstractMultiworkerIOReactor.java:351) ~[org.opensearch_opensearch-spark-standalone_2.12-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at org.apache.http.impl.nio.conn.PoolingNHttpClientConnectionManager.execute(PoolingNHttpClientConnectionManager.java:221) ~[org.opensearch_opensearch-spark-standalone_2.12-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at org.apache.http.impl.nio.client.CloseableHttpAsyncClientBase$1.run(CloseableHttpAsyncClientBase.java:64) ~[org.opensearch_opensearch-spark-standalone_2.12-0.1.0-SNAPSHOT.jar:0.1.0-SNAPSHOT]
	at java.lang.Thread.run(Thread.java:833) ~[?:?]
23/10/27 06:06:14 INFO SparkUI: Stopped Spark web UI at http://[2a05:d018:107:5a00:db96:4136:4e9a:f757]:4040

```


**Do you have any additional context?**
resulting error (EMR) 
```
```

--- a/spark-sql-application/src/main/scala/org/apache/spark/sql/FlintJobExecutor.scala
+++ b/spark-sql-application/src/main/scala/org/apache/spark/sql/FlintJobExecutor.scala
@@ -28,7 +28,9 @@ trait FlintJobExecutor {
   def createSparkConf(): SparkConf = {
     new SparkConf()
       .setAppName(getClass.getSimpleName)
-      .set("spark.sql.extensions", "org.opensearch.flint.spark.FlintPPLSparkExtensions,org.opensearch.flint.spark.FlintSparkExtensions")
+      .set(
+        "spark.sql.extensions",
+        "org.opensearch.flint.spark.FlintPPLSparkExtensions,org.opensearch.flint.spark.FlintSparkExtensions")
   }
 
   def createSparkSession(conf: SparkConf): SparkSession = {

--- a/spark-sql-application/src/main/scala/org/apache/spark/sql/FlintJobExecutor.scala
+++ b/spark-sql-application/src/main/scala/org/apache/spark/sql/FlintJobExecutor.scala
@@ -28,7 +28,7 @@ trait FlintJobExecutor {
   def createSparkConf(): SparkConf = {
     new SparkConf()
       .setAppName(getClass.getSimpleName)
-      .set("spark.sql.extensions", "org.opensearch.flint.spark.FlintSparkExtensions")
+      .set("spark.sql.extensions", "org.opensearch.flint.spark.FlintPPLSparkExtensions,org.opensearch.flint.spark.FlintSparkExtensions")
   }
 
   def createSparkSession(conf: SparkConf): SparkSession = {

